src/ai/backend/agent/server.py

```diff
@@ -838,10 +853,10 @@ def main(
     config.override_with_env(raw_cfg, ("container", "bind-host"), "BACKEND_BIND_HOST_OVERRIDE")
     config.override_with_env(raw_cfg, ("container", "sandbox-type"), "BACKEND_SANDBOX_TYPE")
     config.override_with_env(raw_cfg, ("container", "scratch-root"), "BACKEND_SCRATCH_ROOT")
-    if debug:
-        config.override_key(raw_cfg, ("debug", "enabled"), True)
-        config.override_key(raw_cfg, ("logging", "level"), "DEBUG")
-        config.override_key(raw_cfg, ("logging", "pkg-ns", "ai.backend"), "DEBUG")
+
+    config.override_key(raw_cfg, ("debug", "enabled"), log_level.value == "debug")
```

Comment:
Since we're now utilizing `LogSeverity` to represent log level, please replace all occurrences of raw "debug" string with `LogSeverity.DEBUG`.
```suggestion
    config.override_key(raw_cfg, ("debug", "enabled"), log_level== LogSeverity.DEBUG)
```